src/frontend/src/env/networks.icrc.env.ts

```diff
@@ -414,7 +414,15 @@ export const ICRC_TOKENS: IcCkInterface[] = [
 	...(nonNullish(CKWBTC_IC_DATA) ? [CKWBTC_IC_DATA] : []),
 	...(nonNullish(CKUSDT_IC_DATA) ? [CKUSDT_IC_DATA] : []),
 	...(nonNullish(CKWSTETH_IC_DATA) ? [CKWSTETH_IC_DATA] : [])
-];
+].map((token) => ({
```

Comment:
It might be a bit more verbose, but I feel like it would be cleaner not to apply the map here. By adding a map, we are mixing building a list of defined tokens with applying some logic, while the former is less prone to error and more portable.

Reply:
Yes, it makes sense. In the last commit I moved it to a specific mapping function outside that list